R/pkg/inst/tests/testthat/jarTest.R

```diff
@@ -16,17 +16,17 @@
 #
 library(SparkR)
 
-sparkR.session()
+sc <- sparkR.session()
```

Comment:
If we remove `sc` then it prints a line 
```
Java ref type org.apache.spark.sql.SparkSession id 1
```
It doesn't matter in a test case but I wonder if we should return something else / print something better for users who might call this.